Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "chronicle"
-version = "0.1.0"
```

Comment:
why that?

Reply:
I want to pre-release some Docker images to the GitHub container registry and I want to make sure that no one actually tries to use them 😁

Reply:
Does this really determine the docker version?

Reply:
I will create tags on GitHub that will build the Docker image automatically (with the tags as semver)